src/Administration/Resources/app/administration/src/app/component/form/sw-text-field/index.js

```diff
@@ -21,6 +21,12 @@ Component.register('sw-text-field', {
     ],
 
     props: {
+        type: {
```

Comment:
I would prefer to create a new field for this which also extends from the normal field. Then it can be used like this:
```
<sw-field type="email"></sw-field>
```

This would be more consistent :)

Reply:
I added a custom component for the email-type
The input is added to `sw-field` but I decided that the direct access via `sw-email-field` would be nicer for this :)